api-v3-beta.rst

```diff
@@ -0,0 +1,62 @@
+.. post:: August 8, 2019
+   :tags: api, feature, reference
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+
+      Announcing API v3 Beta as invite-only status. Help us to make it *even* better!
+
+========================
+ Announcing API v3 Beta
+========================
+
+In the last months, we have been working on making our API better.
+Considering the limitations of our current REST API v2,
+we decided to make a bigger step forward and create our new shiny API v3 with better and bigger features,
+putting the focus on the user cases we heard about from existing users.
+
+Compared to API v2, our new API v3 has some big differences that make it more user-friendly and useful.
+
+These differences includes:
+
+* API access requires authentication
+* Token based API authentication
+* Read and write
+* Resource lookup can use project's slug
+
+Some examples of new queries and actions that will be possible with the API are:
+
+* `Import a new project`_
+* `List all my projects`_
+* `Activate a version`_
+* `Trigger a build for a specific version`_
+* `Check build status`_
+* `Manage redirects`_ and `environment variables programmatically`_
+
+.. _Import a new project: https://docs.readthedocs.io/page/api/v3.html#project-create
+.. _List all my projects: https://docs.readthedocs.io/page/api/v3.html#projects-list
+.. _Activate a version: https://docs.readthedocs.io/page/api/v3.html#version-update
+.. _Trigger a build for a specific version: https://docs.readthedocs.io/page/api/v3.html#build-triggering
+.. _Check build status: https://docs.readthedocs.io/page/api/v3.html#build-details
+.. _Manage redirects: https://docs.readthedocs.io/page/api/v3.html#redirects
+.. _environment variables programmatically: https://docs.readthedocs.io/page/api/v3.html#environment-variables
+
+If you want to know more about it,
+please check out the full `APIv3 documentation`_.
+
+.. _APIv3 documentation: https://docs.readthedocs.io/page/api/v3.html
+
+
+Invite-only Beta feature
+------------------------
+
+APIv3 is currently in an invite-only beta state on our community site, https://readthedocs.org.
+In case you want to test it and give us feedback on it,
+please `email us`_ to get early access.
```

Comment:
Should we mention any sort of timeline here like when we are going to open beta and finally releasing it?

Reply:
It would good to have that timeline, but we haven't discussed this yet.

I imagine that a 3 months window for this beta testing minimum. Although, we need to see how many early access requests we receive. We haven't heard anything from the people we share some tokens yet :disappointed:

> As a side note, I'd like to open it _before_ implementing "APIv3 Version 3" (https://docs.readthedocs.io/en/latest/development/design/apiv3.html#version-3) and _after_ having "Version 2" all covered.